CRVResponse/src/CrvDigitizer_module.cc

```diff
@@ -79,6 +87,23 @@ namespace mu2e
 
   void CrvDigitizer::produce(art::Event& event) 
   {
+    art::Handle<EventWindowMarker> eventWindowMarker;
+    event.getByLabel(_eventWindowMarkerLabel,"",eventWindowMarker);
+    EventWindowMarker::SpillType spillType = eventWindowMarker->spillType();
+
+    ProditionsHandle<EventTiming> eventTimingHandle;
+    const EventTiming &eventTiming = eventTimingHandle.get(event.id());
+    double TDC0time = eventTiming.timeFromProtonsToDRMarker();
```

Comment:
Here you should be using
```
 const ProtonBunchTimeMC& pbtmc(*pbtmcHandle);
      TDC0time = -1*pbtmc.pbtime_;
```
pbtime is the time of the protons wrt the marker (so -1*pbtime is the marker time wrt to proton time). timeFromProtonsToDRMarker is only the fixed part of pbtime and doesn't include the bunch to bunch jittering. timeFromProtonsToDRMarker should only be used to convert the digitizationTime fcl parameter. (I.e. if timeFromProtonsToDRMarker is 100 ns and digitizationStart is 400 ns, then the cut applied will be 400-100=300 ns after the marker)

Reply:
I changed it here and in Crv reco (where I used ProtonBunchTime instead of ProtonBunchTimeMC).